common_examples/iperf/main/idf_component.yml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+  espressif/ethernet_init:
+    override_path: '../../../ethernet_init/'
+  espressif/iperf-cmd: "^0.1.1"
+  espressif/iperf: "^0.1.1"
```

Comment:
`iperf-cmd` component already has dependency on `iperf` component. So it may not be needed to add it.